.goreleaser.yml

```diff
@@ -80,6 +79,48 @@ nfpms:
       postinstall: ./docs/packaging/postinstall.sh
       postremove: ./docs/packaging/postremove.sh
 
+kos:
+  - id: ghcr
+    repository: ghcr.io/juanfont/headscale
+    base_image: gcr.io/distroless/base-debian11
+    build: headscale
+    main: ./cmd/headscale
+    env:
+      - CGO_ENABLED=0
+    platforms:
+      - linux/amd64
+      - linux/386
+      - linux/arm64
+      - linux/arm/v7
+      - linux/arm/v6
+      - linux/arm/v5
+    tags:
+      - latest
+      - "{{ .Tag }}"
+      - "{{ .Major }}.{{ .Minor }}.{{ .Patch }}"
+      - "{{ .Major }}.{{ .Minor }}"
+      - "{{ .Major }}"
```

Comment:
I would also try to use the same tags as we have until now. So I would be missing the sha tag. Like [this](https://github.com/juanfont/headscale/pkgs/container/headscale/156430085?tag=sha-6049ec7)
It seems like that goreleaser does not support sha256 sums to tag the containers with. [templating](https://goreleaser.com/customization/templates/#common-fields)

We could maybe replace it with the short commit hash (sha1).

```
- "sha-{{ .ShortCommit }}"
```

The longer tags with the whole sha256-sum won't be possible anymore, but to be honest most people didn't downloaded them anyway.